web/src/components/data-fields/IndustryDropdown.tsx

```diff
@@ -55,9 +56,10 @@ export const IndustryDropdown = (props: Props): ReactElement => {
     let homeBasedBusiness: boolean | undefined = undefined;
     let cannabisLicenseType = undefined;
 
-    if (!isHomeBasedBusinessApplicable(industryId)) {
-      homeBasedBusiness = false;
-    }
+    homeBasedBusiness =
+      isHomeBasedBusinessApplicable(industryId) || !nexusLocationInNewJersey(state.profileData)
+        ? homeBasedBusiness
+        : false;
```

Comment:
dust - I think this could be more readable?

`homeBasedBusiness =
      isHomeBasedBusinessApplicable(industryId) || !nexusLocationInNewJersey(state.profileData)
        ? undefined
        : false;`

maybe? Because it homeBaseBusiness will always be undefined at this point?

Alternatively you could do an if because the if case isn't actually changing the value of homeBasedBusiness. 

Super nit picky on my end. Feel free to ignore

Reply:
Also this is a bit of change in logic isn't it? I didn't see something about this in the ticket initially, but I assume you talked to someone and they wanted it right?

Reply:
I refactored it back to what I had initially/you suggested because the logic is different when written this way.